types/office-js-preview/index.d.ts

```diff
@@ -539,7 +540,9 @@ declare namespace Office {
          */
         AppointmentTimeChanged,
         /**
-         * Occurs when an attachment is added to or removed from an item. **Important**: This event can only be handled in a task pane. It isn't supported by function commands.
+         * Occurs when an attachment is added to or removed from an item.
+         *
+         * **Important**: This event can only be handled in a task pane. It isn't supported by function commands.
```

Comment:
Isn't this true of all events? I don't see how a function that's triggered by a ribbon button can handle an event. Or is this trying to say that a function command can't _register_ a handler for an event?

Reply:
It refers to the latter. I'll update the wording to make this clearer.